src/puya/awst/nodes.py

```diff
@@ -199,6 +221,16 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 
 @attrs.frozen
 class IfElse(Statement):
+    """
+    A stand c-like if-then-else statement. Execution will continue to the following statement after
+    an if_branch or else_branch has been executed unless those branches contain a control op which
+    leads elsewhere (eg. return/break/continue)
+
+    condition: A boolean condition to be evaluated
```

Comment:
Field documentation should go on the fields - they can't technically be docstrings but most tooling (e.g. VSCode and PyCharm) will treat them as such.